tests/CacheCompat/CommonCache.Test.Unit/CommonCache.Test.Unit.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
-    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.0" />
```

Comment:
I updated all of these because VS started complaining about tests not found.

Reply:
should this be done for all projects?

Reply:
Yes, it should. And it has been done :) - at least for all .net test projects. I did not touch the xamarin stuff because App Center might not like newer version of stuff.

Reply:
interesting... I only saw this done for one project in my initial look (properly switched to iteration 1)

---
In reply to: [389798149](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1689#discussion_r389798149) [](ancestors = 389798149)